bullet_train-themes-tailwind_css/app/views/themes/tailwind_css/attributes/_attempt.html.erb

```diff
@@ -6,6 +6,8 @@
   <span class="inline-block">
     <% if object.send(success_method) %>
       <i class="fal fa-check ti ti-check text-green-500"></i>
+    <% elsif object.send(failure_method) %>
```

Comment:
I moved that up because the `still_attempting?` method will return true even after the delivery time has expired.